packages/web/src/flash/Flash.tsx

```diff
@@ -1,24 +1,30 @@
 import { useEffect, useState } from 'react'
 
 import { useFlash } from 'src/flash/FlashContext'
+import { FlashMessage } from 'src/flash/FlashReducer'
```

Comment:
Isn't a type-only import enough here? And if not, why? (If you don't mind explaining 🙂

Reply:
That was how I did it first, then some accident happened. Will fix.